src/database/migrations/20190704115631-create-users.js

```diff
@@ -0,0 +1,59 @@
+module.exports = {
+  up: (queryInterface, Sequelize) => {
```

Comment:
Good work so far @codexempire I think you should remove this migration and add/remove specific columns you intend to add/remove specifically.